S3/locals.tf

```diff
@@ -2,6 +2,7 @@ locals {
   common_tags = {
     (var.billing_tag_key)  = var.billing_tag_value
     Terraform              = "true"
+    Log_Bucket             = lookup(var.tags, "Log_Bucket", "false")
```

Comment:
Shouldn't this just default to false all the time, when would we want it to be true?

Reply:
Well, there was an instance where the [S3 bucket module](https://github.com/cds-snc/salesforce-backup/blob/3db3d0e0ba330ec38a4cb9354b26799419029778/terragrunt/s3.tf#L19) was used to create the target bucket (which is for logs as far as I understand based on [documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#target_bucket)) instead of the log bucket module.

Reply:
I'll have to change that log bucket